### PR TITLE
Jetpack connect: Jetpack branding

### PR DESCRIPTION
**Pull request by sirreal**
Add the logo to the top of the Jetpack connect main wrapper.
Make the primary button in Jetpack connect green.

Plans page is unaffected, header text changes in another pr (#22523)

see p1HpG7-4Mw-p2

### Button states

I've updated the primary button states based on some existing work at https://wordpress.com/jetpack/new —
 7b8e637

I've also updated the button there to leverage the styles defined here which are slightly different using `@extend`.

#### Disabled

disabled

#### Hover

hover

#### Focus

focus

### Site entry (`/jetpack/connect`)

This wasn't included in designs, but I assume that was an oversight and included it here.

#### Before

b-entry

#### After

a-entry

#### Before

b-signup

#### After

a-signup

#### Before

b-auth

#### After

a-auth

## Testing

Go through the flow and validate designs (p1HpG7-4Mw-p2). Changes are visual (no functional changes) and only available in development and wpcalypso environments. They can be seen on [calypso.live](https://calypso.live/jetpack/connect?branch=update/jetpack/branded-logo-jpc)

### CODE REVIEW

File: client/jetpack-connect/jetpack-new-site/style.scss [hunk 1]
```diff
@@ -140,24 +140,8 @@
...
-			box-shadow: 0 0 0 2px lighten( $green-jetpack, 25% );
-		}
 
-		&[disabled],
-		&:disabled {
-			background: tint( $green-jetpack, 50% );
-			border-color: tint( $green-jetpack, 35% );
-			color: $white;
-		}
+		@extend .jetpack-connect__main.jetpack-branded .button.is-primary;
```
Comment (tyxla) [inline client/jetpack-connect/jetpack-new-site/style.scss]: Weird stuff, SASS can't compile this one and I'm getting a "can't extend nested selectors" error:

Comment (MichaelArestad) [reply client/jetpack-connect/jetpack-new-site/style.scss]: Let's not use `@extend` at all in Calypso. It's technically against guidelines and it's not efficient for a few reasons.

**Comment by joanrho** [comment]
@sirreal I couldn't get this branch to build, but based on the screenshots, the design changes look good. Thanks for catching the extra screens!

File: client/jetpack-connect/jetpack-new-site/style.scss (hunk 1 above)
Comment (sirreal) [reply client/jetpack-connect/jetpack-new-site/style.scss]: 👍 I removed the `@extend` in 0aa36092789a4c2f3de7c3b7bc85b6fde4d6d2aa

This builds correctly now. I don't know how I missed that 😕

**Comment by sirreal** [comment]
> could we change the copy of the button from 'sign up and connect' to 'Create your account'

👍 @rickybanister Went ahead and made that change in d319ec62b0f8a697d912ff4fa30bbc7299575dd6
, here's the signup step now:

signup

**Comment by sirreal** [comment]
Sounds like the design is all set. I've fixed the build issue 🙇 😬  and this should be ready to go after code review.

File: client/jetpack-connect/jetpack-new-site/style.scss [hunk 2]
```diff
@@ -90,7 +90,7 @@
 }
 
 .jetpack-new-site__wpcom-site {
-	
+
```
Comment (tyxla) [inline client/jetpack-connect/jetpack-new-site/style.scss]: I'd go for removing this line completely.

**Comment by tyxla** [comment]
Rebased.

**Pull request merged**